### PR TITLE
Added a quote in quotes.json

### PR DESCRIPTION
**Pull request by AlacritousCreature**
## ✅ New Quote Checklist
- [ ] The quote(s) added are in alphabetical order and not ❌ duplicates.
- [ ] The quote(s) added are about **learning a foreign language**.
- [ ] The sentence attribute does not have inner quotation marks, and is ended with a period.
- [ ] Copy pasted the raw JSON `quotes.json` file to [JSONLint](https://jsonlint.com/) and made sure that the modified JSON file is valid.

## ✅ New Language Checklist
- [ ] There is ❌ no assigned issue or pull request for this new language.
- [ ] ➕ Added a new JSON object in the `server/data/languages.json` file.
- [ ] Added the **voice** 🎤 attribute supported by ResponsiveVoice API.
- [ ] Added the **flag** 🏳 atrribute in the format of the twemoji-awesome CSS syntax.
- [ ] Added the **romanization** atrribute for appropriate languages.
- [ ] ➕ Created the `{language.json}` file inside the `server/data/words` folder containing the list of the 3000 words.
- [ ] Removed duplicates and noted 🗒 them in the Pull Request.
- [ ] Increment `currentLanguagesCount` in `client/src/store/index.js`
- [ ] Add new language in `README.md`

### CODE REVIEW

**Comment by AlacritousCreature** [comment]
> Hi! Thanks for contributing but the code should be about learning a foreign language

Oh..sorry, i will change it.

**Comment by AlacritousCreature** [comment]
hey! I have made the changes in the pull request. Please review it.

File: server/data/quotes.json [hunk 1]
```diff
@@ -202,7 +202,7 @@
     {
         "author": "Haruki Murakami",
         "sentence": "Learning another language is like becoming another person."
-    },
+    },
```
Comment (jayehernandez) [inline server/data/quotes.json:205]: Thanks for changing the quote! Please remove the whitespace here, and I'll merge right after.

Comment (AlacritousCreature) [reply server/data/quotes.json]: Thanks for reviewing, I have removed the white space on line 205 of quotes.json

**Comment by jayehernandez** [comment]
@all-contributors please add @AlacritousCreature for content

**Comment by AlacritousCreature** [comment]
Thanks for helping in my first contributions!

**Pull request merged**